index.styl

```diff
@@ -0,0 +1,60 @@
+display
+position
+top
+right
+bottom
+left
+
+float
+clear
+flex-*
+(align-*, justify-*, order)
+
+z-index
+opacity
+visibility
+
+width
+max-width
+height
+max-height
+
+margin
+border
+padding
+
+overflow
+box-sizing
+vertical-align
+
+outline
+background
+box-shadow
+
+// element- and type-specific styles should be separated
+list-style
+caption-side
+
+table-layout
+border-collapse
+border-spacing
+empty-cells
+
+text-align
+text-transform
+text-decoration
+text-shadow
+
+font-family
+font-size
+font-weight
+
+color
```

Comment:
@kenworthyc i've always put color next to background so that i know theres no glaring contrast issues as it also affects alt text color. to me it is a more fundamental declaration since it affects accessibility. thoughts?